### PR TITLE
Upgrade to Filebeat 8.3.3, add workflow for build and push

### PR DESCRIPTION
**Pull request by mirii1994**
- Upgrade from filebeat 7.12 to filebeat 8.3.3.
- Add workflow for building & pushing docker images.
- Update changelog.
- Remove from script print of version (unnecessary and prone to mistakes).

### CODE REVIEW

File: filebeat-yml-script.py [hunk 1]
```diff
@@ -33,6 +32,7 @@ def get_log_level():
 logzio_region = os.getenv("LOGZIO_REGION", "")
 logzio_codec = os.getenv("LOGZIO_CODEC", "plain").lower()
 logzio_codec_list = ["plain", "json"]
+input_encoding = os.getenv("INPUT_ENCODING", "utf-8")
```
Comment (Doron-Bargo) [inline filebeat-yml-script.py:35]: Please add a check that the customer set the right encoding
Also please keep in mind - big /small characters

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
Hi, please see my comment

File: filebeat-yml-script.py (hunk 1 above)
Comment (mirii1994) [reply filebeat-yml-script.py]: Done.

**Review by Doron-Bargo — APPROVED** [review]
lgtm

**Pull request merged**